src/statement.ts

```diff
@@ -0,0 +1,240 @@
+import {
```

Comment:
GO HERE NEXT: This is actually the workhorse API of the entire thing now. This is where you "just write SQL". It is a really great API (not mine) and I find it really nice to work with. It took some deep thinking to make it fit a web3 world where we have to wait for transactions and things, but in the end, I think it is really intuitive! I hope you like it!